src/renderer/components/dock/log-list.tsx

```diff
@@ -79,12 +81,15 @@ export class LogList extends React.Component<Props> {
   @computed
   get logs() {
     const showTimestamps = logTabStore.getData(this.props.id).showTimestamps;
+    const { preferences } = userStore;
 
     if (!showTimestamps) {
       return logStore.logsWithoutTimestamps;
     }
 
-    return this.props.logs;
+    return this.props.logs.map(log => (
+      `${moment.tz(logStore.getTimestamps(log)[0] || "", preferences.localeTimezone).format()} ${logStore.removeTimestamps(log)}`
+    ));
```

Comment:
I think that logStore should have a function "splitOutTimestamp" which returns the "timestamp" string and the "rest" string. Would you mind adding that?

Reply:
done